app/src/lib/editors/launch.ts

```diff
@@ -30,7 +30,8 @@ export async function launchExternalEditor(
   }
 
   if (editor.usesShell) {
-    spawn(`"${editorPath}"`, [`"${fullPath}"`], { ...opts, shell: true })
+    const args = editor.shellArgs ? editor.shellArgs : []
```

Comment:
This can be simplified as:
```suggestion
    const args = editor.shellArgs ?? []
```

Reply:
sorted in last commit